test/unit/common/persistence/PersistenceManagerTest.php

```diff
@@ -109,3 +117,11 @@ protected function getSqlConfig()
         ];
     }
 }
+
+class UnitTestService implements SchemaProviderInterface
```

Comment:
I think it would be more elegant using an anonymous class directly instantiated in the test itself.

Reply:
done